documentation/adoc/appendix_metrics.adoc

```diff
@@ -0,0 +1,131 @@
+[appendix]
+== Metrics
+
+This section describes how to deploy a Prometheus server for scraping metrics from the Kafka cluster and showing them using a Grafana dashboard. The resources provided are examples to show how Kafka metrics can be stored in Prometheus: They are not a recommended configuration, and further support should be available from the Prometheus and Grafana communities.
+
+When using `minikube` or `minishift` for your Apache Kafka deployment adding Prometheus and Grafana servers, it would be better to increase the default amount of memory available to the virtual machine (i.e. not using the default 2 GB of RAM but 4 GB instead).
+
+Information on how to increase the default amount of memory can be found in the following section <<_installing_kubernetes_and_openshift_cluster>>
+
+=== Deploying on OpenShift
+
+==== Prometheus
+
+The Prometheus server configuration uses a service discovery feature in order to discover the pods in the cluster from which it gets metrics.
+In order to have this feature working, it's necessary to provide access to the API server to get the pod list to the service account used for running the Prometheus service pod (the service account `prometheus-server` is used).
+
+[source]
+export NAMESPACE=[namespace]
+oc login -u system:admin
+oc create sa prometheus-server
+oc adm policy add-cluster-role-to-user cluster-reader system:serviceaccount:${NAMESPACE}:prometheus-server
+oc login -u developer
+
+where `[namespace]` is the namespace/project where the Apache Kafka cluster was deployed.
+
+After that, it's possible to deploy the Prometheus server in the cluster :
+
+. Create the https://github.com/strimzi/strimzi/blob/master/metrics/examples/prometheus/openshift-template.yaml[provided "prometheus" template] by running
++
+[source]
+oc create -f metrics/examples/prometheus/openshift-template.yaml
+
+. Create a new app using the "prometheus" template:
++
+[source]
+oc new-app prometheus
```

Comment:
Why do we use the template here? It just adds more bloat which we have to maintain. I would suggest to remove the tem,plate and use the Kubernetes YAML file for OpenSHift.

Reply:
@ppatierno you are the creator of the first sample of this document. Do you agree with this statement? or what would you recommend?

Reply:
@serrss agree with @scholzj . We can get rid of the OpenShift template.

Reply:
@scholzj @ppatierno removed OpenShift template and updated links to Kubernetes YAML